MAINTAINERS.md

```diff
@@ -0,0 +1,145 @@
+| ------------------------------------------------- |
+| [Christian Tzolov](https://github.com/tzolov)     |
+| [Dariusz Jędrzejczyk](https://github.com/chemicL) |
+
+### Ruby SDK
+
+| Name                                               |
+| -------------------------------------------------- |
+| [Topher Bullock](https://github.com/topherbullock) |
+| [Koichi Ito](https://github.com/koic)              |
+| [Ateş Göral](https://github.com/atesgoral)         |
+
+### Swift SDK
+
+| Name                                           |
+| ---------------------------------------------- |
+| [Mattt Zmuda](https://github.com/mattt)        |
+| [Carl Peaslee](https://github.com/carlpeaslee) |
+
+### Go SDK
+
+| Name                                       |
+| ------------------------------------------ |
+| [Rob Findley](https://github.com/findleyr) |
+
+### C# SDK
+
+| Name                                           |
+| ---------------------------------------------- |
+| [Stephan Halter](https://github.com/halter73)  |
+| [Mike Kistler](https://github.com/mikekistler) |
+
+### Kotlin SDK
+
+| Name                                         |
+| -------------------------------------------- |
+| [Leonid Stashevsky](https://github.com/e5l)  |
+| [Sergey Ignatov](https://github.com/ignatov) |
+
+### Python SDK
+
+| Name                                             |
+| ------------------------------------------------ |
+| [Inna Harper](https://github.com/ihrpr)          |
+| [Jerome Swannack](https://github.com/jerome3o)   |
+| [Samuel Colvin](https://github.com/samuelcolvin) |
+| [Marcelo Trylesinski](https://github.com/Kludex) |
+
+### TypeScript SDK
+
+| Name                                                   |
+| ------------------------------------------------------ |
+| [Inna Harper](https://github.com/ihrpr)                |
+| [Felix Weinberger](https://github.com/felixweinberger) |
+| [Olivier Chafik](https://github.com/ochafik)           |
+
+### Rust SDK
+
+| Name                                           |
+| ---------------------------------------------- |
+| [Alex Hancock](https://github.com/alexhancock) |
+
+## Project Maintainers
+
+| Name                                                 | Project                         |
+| ---------------------------------------------------- | ------------------------------- |
+| [Glen Maddern](https://github.com/geelen)            | use-mcp                         |
+| [Ola Hungerford](https://github.com/olaservo)        | Inspector<br/>Reference Servers |
+| [Cliff Hall](https://github.com/cliffhall)           | Inspector<br/>Reference Servers |
+| [Toby Padilla](https://github.com/toby)              | Registry                        |
+| [Tadas Antanavicius](https://github.com/tadasant)    | Registry<br/>Reference Servers  |
+| [Shaun Smith](https://github.com/evalstate)          | Reference Servers               |
+| [Jonathan Hefner](https://github.com/jonathanhefner) | Reference Servers               |
```

Comment:
In this case, it could be expressed with an annotation like the following.

```suggestion
- [Glen Maddern](https://github.com/geelen) (use-mcp)
- [Ola Hungerford](https://github.com/olaservo) (Inspector and Reference Servers)
- [Cliff Hall](https://github.com/cliffhall) (Inspector and Reference Servers)
- [Toby Padilla](https://github.com/toby) (Registry)
- [Tadas Antanavicius](https://github.com/tadasant) (Registry and Reference Servers)
- [Shaun Smith](https://github.com/evalstate) (Reference Servers)
- [Jonathan Hefner](https://github.com/jonathanhefner) (Reference Servers)
```